pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -224,6 +224,45 @@ public void setUp()
 
     // Wait for all documents loaded
     waitForAllDocsLoaded(600_000L);
+
+    // Try to reload all the segments with force download from the controller URI.
+    reloadAllSegments(TEST_UPDATED_INVERTED_INDEX_QUERY, true, getCountStarResult());
+
+    // Try to upload all the segments again with force download from the controller URI.
+    try {
+      uploadSegments(getTableName(), tarDirs);
+    } catch (Exception e) {
+      // If enableParallelPushProtection is enabled and the same segment is uploaded concurrently, we could get one
+      // of the two exception - 409 conflict of the second call enters ProcessExistingSegment ; segmentZkMetadata
+      // creation failure if both calls entered ProcessNewSegment. In/such cases ensure that we upload all the
+      // segments again/to ensure that the data is setup correctly.
```

Comment:
looks like the comment is copied from L205 above, but there seems some typos `In/such... again/to...`

Reply:
updated.